### PR TITLE
Windows and vcpkg

### PR DESCRIPTION
**Pull request by jeremad**

### CODE REVIEW

File: IO/src/av/CMakeLists.txt [hunk 1]
```diff
@@ -58,15 +58,12 @@ function(setup_av_plugin PLUGIN_NAME BACKEND_NAME USE_CUDA)
...
   set_property(TARGET ${PLUGIN_NAME} PROPERTY CXX_STANDARD 14)
 
-  set(FFMPEG_INCLUDE_PATH ${CMAKE_EXTERNAL_DEPS}/include/ffmpeg)
   if(MACPORTS)
     set(FFMPEG_INCLUDE_PATH /opt/local/include)
   else()
     set(FFMPEG_INCLUDE_PATH /usr/local/include)
   endif()
 
-  target_include_directories(${PLUGIN_NAME} SYSTEM PRIVATE ${FFMPEG_INCLUDE_PATH})
```
Comment (w-m) [inline IO/src/av/CMakeLists.txt]: This is needed when building Mac with MACPORTS currently (see line 63). Can we keep the line in?

Comment (jeremad) [reply IO/src/av/CMakeLists.txt]: sure, my ax was a bit too sharpe

File: IO/src/decklink/src/decklink_writer.cpp [hunk 2]
```diff
@@ -109,7 +109,7 @@ DeckLinkWriter* DeckLinkWriter::create(const Ptv::Value* config, const std::stri
     }
     std::shared_ptr<IDeckLinkConfiguration> configuration(tempConfiguration,
                                                           VideoStitch::DeckLink::getDefaultDeleter());
-    configuration->SetFlag(bmdDeckLinkConfigUse1080pNotPsF, true);
+    configuration->SetFlag(bmdDeckLinkConfigCapture1080pAsPsF, true);
```
[the remark this answers is not in the document]
Comment (jeremad) [reply IO/src/decklink/src/decklink_writer.cpp]: I forgot to set at false:
https://forum.blackmagicdesign.com/viewtopic.php?f=12&t=64372

[the remark this answers is not in the document]
Comment (w-m) [reply IO/src/decklink/src/decklink_writer.cpp]: Did you commit the fix somewhere already?

[the remark this answers is not in the document]
Comment (jeremad) [reply IO/src/decklink/src/decklink_writer.cpp]: It seems I did not, let me push force

File: IO/src/av/CMakeLists.txt (hunk 1 above)
Comment (jeremad) [reply IO/src/av/CMakeLists.txt]: fixed

**Pull request merged**